components/logicbutton/stories/template.js

```diff
@@ -1,20 +1,18 @@
 import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
-// import { ifDefined } from 'lit-html/directives/if-defined.js';
 
 import "../index.css";
 
 export const Template = ({
   rootClass = "spectrum-LogicButton",
   customClasses = [],
-  size = "m",
-  // ...globals
+  variant = 'and'
 }) => {
   return html`
-    <div class=${classMap({
+    <button class=${classMap({
```

Comment:
Certainly not required but you could include the `is-disabled` and `is-focused` states as well.

Reply:
I'm going to add these, it's a good suggestion, thanks!